src/view/waypoint-form-view.js

```diff
@@ -1,175 +1,132 @@
+
+             <div class="event__type-list">
+               <fieldset class="event__type-group">
+                 <legend class="visually-hidden">Event type</legend>
+                 ${createEventTypeItemTemplate(WAYPOINT_TYPES)}
+               </fieldset>
+             </div>
+           </div>
+
+           <div class="event__field-group  event__field-group--destination">
+             <label class="event__label  event__type-output" for="event-destination-1">
+               ${type ? capitalize(type) : ''}
+             </label>
+             <input class="event__input  event__input--destination" id="event-destination-1" type="text" name="event-destination" value="${destination ? destination.name : ''}" list="destination-list-1">
+             <datalist id="destination-list-1">
+               ${createCityTemplate(CITIES)}
+             </datalist>
+           </div>
+
+           <div class="event__field-group  event__field-group--time">
+             <label class="visually-hidden" for="event-start-time-1">From</label>
+             <input class="event__input  event__input--time" id="event-start-time-1" type="text" name="event-start-time" value="${startTimeInForm}">
+             &mdash;
+             <label class="visually-hidden" for="event-end-time-1">To</label>
+             <input class="event__input  event__input--time" id="event-end-time-1" type="text" name="event-end-time" value="${endTimeInForm}">
+           </div>
+
+           <div class="event__field-group  event__field-group--price">
+             <label class="event__label" for="event-price-1">
+               <span class="visually-hidden">Price</span>
+               &euro;
+             </label>
+             <input class="event__input  event__input--price" id="event-price-1" type="text" name="event-price" value="${basePrice}">
+           </div>
+
+           <button class="event__save-btn  btn  btn--blue" type="submit">Save</button>
+           <button class="event__reset-btn" type="reset">Cancel</button>
+         </header>
+         <section class="event__details">
+           <section class="event__section  event__section--offers">
+             <h3 class="event__section-title  event__section-title--offers">Offers</h3>
+
+             <div class="event__available-offers">
+               ${offers ? createOfferTemplate(offers) : ''}
+             </div>
+           </section>
+
+           <section class="event__section  event__section--destination">
+             <h3 class="event__section-title  event__section-title--destination">Destination</h3>
+             <p class="event__destination-description">${destination ? destination.description : ''}</p>
+
+             <div class="event__photos-container">
+               <div class="event__photos-tape">
+                 ${destination ? createPictureTemplate(destination.pictures) : ''}
+               </div>
+             </div>
+           </section>
```

Comment:
в принципе если destination не пришёл , то можно всю эту секцию не отрисовывать
```suggestion

        ${ destination ? 
           `<section class="event__section  event__section--destination">
             <h3 class="event__section-title  event__section-title--destination">Destination</h3>
             <p class="event__destination-description">${destination.description}</p>

             <div class="event__photos-container">
               <div class="event__photos-tape">
                 ${createPictureTemplate(destination.pictures)}
               </div>
             </div>
           </section>` : ''
           }
```